specification/app/resource-manager/Microsoft.App/preview/2024-10-02-preview/ContainerApps.json

```diff
@@ -667,6 +667,11 @@
           "description": "Optional. Max inactive revisions a Container App can have.",
           "type": "integer"
         },
+        "revisionTransitionThreshold": {
+          "format": "int32",
```

Comment:
You can add minimum and maximum property

Reply:
added to description

Reply:
Not blocking approval on this, but why not add "minimum": 0, "maximum": 100, "default": 100, so it's part of the spec?